docs/source/reference_aggregation.rst

```diff
@@ -0,0 +1,191 @@
+Other Methods of Aggregations
+=============================
+
+As we have already seen in this `page <https://fsspec.github.io/kerchunk/test_example.html#multi-file-jsons>`_,
+that the main purpose of ``kerchunk`` it to generate references, to view whole archive of files like
+GRIB2, NetCDF etc. allowing us for *in-situ* access to the data. In this part of the documentation,
+we will see some other efficient ways of combining references.
+
+GRIB Aggregations
+-----------------
+
+This new method for reference aggregation, developed by **Camus Energy**, is based on GRIB2 files. Utilizing
+this method can significantly reduce the time required to combine references, cutting it down to
+a fraction of the previous duration. In reality, this approach builds upon consolidating references
+with ``kerchunk.combine.MultiZarrtoZarr``, making it faster. The functions and operations used in this
+will be a part of the ``kerchunk``'s API. You can follow this `video <https://discourse.pangeo.io/t/pangeo-showcase-optimizations-for-kerchunk-aggregation-and-zarr-i-o-at-scale-for-machine-learning/4074>`_
+for the intial discussion.
+
+*How is it faster*
+
+Every GRIB file stored on cloud platforms such as **AWS** and **GCP** is accompanied by its
```

Comment:
Not true! **IF** there are .idx files, we can use them.

Reply:
The grib and idx files existed in pairs, so I mentioned it.